core/tests/gae_suite_test.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for gae_suite."""
+
+import unittest
+
+from core.tests import gae_suite
+from core.tests import test_utils
+
+
+class GaeSuiteTests(test_utils.GenericTestBase):
+
+    def test_cannot_create_test_suites_with_invalid_test_target_format(self):
+        with self.assertRaisesRegexp(
+            Exception, 'The delimiter in test_target should be a dot (.)'):
+            gae_suite.create_test_suites(test_target='core/controllers')
+
+    def test_create_test_suites(self):
+        test_suite = gae_suite.create_test_suites(
+            test_target='core.controllers.admin_test')
```

Comment:
Can we find a way to do this without using another existing test as the target here? (Maybe use this file as the target? Not sure if that will work). Will make it less likely to fail due to unrelated changes!

Reply:
Done!